products/numpy.md

```diff
@@ -4,14 +4,20 @@ category: framework
 iconSlug: numpy
 permalink: /numpy
 releasePolicyLink: https://numpy.org/neps/nep-0029-deprecation_policy.html
+changelogTemplate: https://numpy.org/doc/stable/release/__LATEST__-notes.html
 versionCommand: python -c "import numpy; print(numpy.__version__)"
 releaseDateColumn: true
 auto:
 -   pypi: numpy
 
-# EOL(R) = releaseDate(R)+1yr,1day
+# EOL(R) = releaseDate(R)+1yr+1day
 # But at no point should any of the last 3 releases become unsupported.
 releases:
+-   releaseCycle: "1.24"
+    releaseDate: 2022-12-18
+    latest: "1.24.0"
+    eol: 2023-12-19
```

Comment:
@BiNZGi, why is the EOL date for 1.24 (19 Dec 2023) before the EOL date for 1.23 (24 Jun 2024) ? Is 1.23 some kind of LTS ?

Looks like versions 1.20 to 1.23 used the following formula : `EOL(R) = releaseDate(R)+2yr+1day`

Reply:
Good catch, I was misled by the comment in our doc what was wrong. It was always 2yr. Will fix...